praw/models/listing/mixins/submission.py

```diff
@@ -13,6 +13,17 @@ def duplicates(self, **generator_kwargs):
         Additional keyword arguments are passed in the initialization of
         :class:`.ListingGenerator`.
 
+        Example usage:
+
+        .. code:: python
+
+            submission = reddit.submission(id='5or86n')
+
+            # get a ListingGenerator of duplicate submissions
+            duplicates = submission.duplicates()
```

Comment:
Can you show this example via a `for duplicate in submission.duplicates()` rather than binding the generator to a variable? As-is I think some people who do not understand generators might confuse what `duplicates` represents and think they could treat it as a list.